src/cloud.c

```diff
@@ -59,6 +59,7 @@
 #define AMQP_CMD_DATA_PUBLISH "data.publish"
 #define AMQP_CMD_DEVICE_REGISTER "device.register"
 #define AMQP_CMD_DEVICE_UNREGISTER "device.unregister"
+#define AMQP_CMD_SCHEMA_UPDATE "schema.update"
```

Comment:
About commit message: 3bd3f0c2b61c28391801febfed4815c576909ee4
... so the schema list is sent via amqp instead of websockets.

We can abstract if we are communicating with cloud and fog.

Reply:
Done.